docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -1639,6 +1639,38 @@ spec:
 For more information on how to configure a pod to use a persistent volume for storage in Kubernetes,
 please refer to [here](https://kubernetes.io/docs/tasks/configure-pod-container/configure-persistent-volume-storage/).
 
+### Set up Alluxio Proxy server
```

Comment:
```suggestion
### Start Alluxio Proxy server
```

Reply:
Set up is hard to follow. It will be better to change to sub-steps, like change configurations, start the proxy server, stop proxy server, etc.